### ISSUE TITLE
Save logs and data for each training run, allow model export/import, training continuation, etc.

### ISSUE THREAD

**Issue by ykevu** [vanvalenlab/deepcell-label#415]
Currently, the in-browser training does a single, cancellable run and saves the final model but no other data about the run. This also leads to some visual bugs, like changing the number of epochs on a second run will make the loading bar change size. Rerunning the training will overwrite the first model, and there is no way of exporting the model. The following list of features should eventually be implemented:

- Save all hyperparameter data, the loss logs and confusion matrix, and final model after each run without being overwritten by future training runs
     - Allow one to select previous runs, use those models, and view the visualization data and such
- Pause and resume training, where pausing saves the model
- Continue training a model that was already saved
- Export and import models from previous runs
- More advanced and customizable features, such as asking to save model on every X epochs, early stopping, auto stopping, etc.
